### PR TITLE
Bug/3464 fix login bug

### PR DESCRIPTION
**Pull request by Cal-L**
**Description**

#3464 

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

**Review by gantunesr — APPROVED** [review]
LGTM. Just left some questions and a suggestion.

File: app/components/Views/Login/index.js [hunk 1]
```diff
@@ -448,8 +449,6 @@ class Login extends PureComponent {
 	updateBiometryChoice = async (biometryChoice) => {
 		if (!biometryChoice) {
 			await AsyncStorage.setItem(BIOMETRY_CHOICE_DISABLED, TRUE);
-			// This line will disable biometrics the next time SecureKeychain.getGenericPassword is called
-			await SecureKeychain.resetGenericPassword();
```
Comment (sethkfman) [inline app/components/Views/Login/index.js:452]: I think this should still be included `SecureKeychain.resetGenericPassword()` in some from. Maybe we create another function that just clears the `Keychain.resetGenericPassword(options);`. 

Even if we logout and the password would still be available in the Keychain.

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮 🌮

File: app/components/Views/Login/index.js (hunk 1 above)
Comment (sethkfman) [reply app/components/Views/Login/index.js:452]: Discussed and see where this has been added in a different location

**Pull request merged**